spec/ics-027-interchain-account/README.md

```diff
@@ -0,0 +1,296 @@
+
+### Definitions  
+
+The IBC handler interface & IBC relayer module interface are as defined in [ICS 25](https://github.com/cosmos/ics/tree/master/spec/ics-025-handler-interface) and [ICS 26](https://github.com/cosmos/ics/tree/master/spec/ics-026-routing-module), respectively.  
+
+### Desired Properties  
+
+- Permissionless  
+- Fault containment: Interchain account must follow rules of its host chain, even in times of Byzantine behavior by the counterparty chain (the chain that manages the account)  
+- The chain that controls the account must process the results asynchronously and according to the chain's logic. The result should be 0x0 if the transaction was successful and an error code other than 0x0 if the transaction failed.
+- Sending and receiving transactions will be processed in an ordered channel where packets are delivered exactly in the order which they were sent.  
+
+## Technical Specification  
+
+The implementation of interchain account is non-symmetric. This means that each chain can have a different way to generate an interchain account and deserialize the transaction bytes and a different set of transactions that they can execute. For example, chains that use the Cosmos SDK will deserialise tx bytes using Amino, but if the counterparty chain is a smart contract on Ethereum, it may deserialise tx bytes by an ABI that is a minimal serialisation algorithm for the smart contract.  
+The interchain account specification defines the general way to register an interchain account and transfer tx bytes. The counterparty chain is responsible for deserialising and executing the tx bytes, and the sending chain should know how counterparty chain will handle the tx bytes in advance.
+
+Each chain must satisfy following features to create a interchain account:
+
+- New interchain accounts must not conflict with existing ones.
+- Each chain must keep track of which counterparty chain created each new interchain account.
+
+Also, each chain must know how the counterparty chains serialize/deserialize transaction bytes in order to send transactions via IBC. And the counterparty chain must implement the process of safely exececuting IBC transactions by verifying the authority of the transaction's signers.
+
+The chain must reject the transaction and must not make a state transition in the following cases:
+
+- The IBC transaction fails to be deserialized.
+- The IBC transaction requests for signers that was not made by counterparty chain.
```

Comment:
```suggestion
- The IBC transaction expects signatures which the counterparty chain did not provide.
```

Reply:
@Thunnini Can you apply this?

Reply:
I think that signature is not needed because this is not communication between the user and the chain, but between the chain and the chain. It is possible to determine if this account is the interchain account of the counterparty chain. Signatures are not required for communication between contracts on Ethereum. I think this is the same principle.

Reply:
Oh, do you mean "requests a signer other than the interchain account" ?

Reply:
I mean that "requests for signers that was not interchain account made by counterparty chain."

Reply:
Makes sense. Updated my suggestion to clarify.